### PR TITLE
added zend-guard-loader support

### PR DESCRIPTION
**Pull request by gsc-dev**

### CODE REVIEW

File: conf/php.ini [hunk 1]
```diff
@@ -1,2 +1,7 @@
-memory_limit = PHP_MEMORY_LIMIT
```
Comment (markshust) [inline conf/php.ini:1]: @gsc-dev can you set this back to `PHP_MEMORY_LIMIT`? this var is set at runtime and can be overridden from docker-compose.yml file.

File: conf/php.ini [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+memory_limit = 1G
 date.timezone = "UTC"
+
+;; Zend Loader Configuration
+zend_extension = /usr/local/etc/php/zend/opcache.so
+zend_extension = /usr/local/etc/php/zend/ZendGuardLoader.so
+always_populate_raw_post_data = -1
```
Comment (markshust) [inline conf/php.ini:7]: You can remove the `always_populate_raw_post_data` line, this is deprecated as of php 5.6.0 http://php.net/manual/en/ini.core.php

File: Dockerfile [hunk 3]
```diff
@@ -34,6 +34,7 @@ ENV PHP_PM_MIN_SPARE_SERVERS 2
 ENV PHP_PM_MAX_SPARE_SERVERS 6
 ENV APP_MAGE_MODE default
 
+COPY lib/* /usr/local/etc/php/zend
```
Comment (markshust) [inline Dockerfile:37]: Add trailing slash to `/usr/local/etc/php/zend` for consistency with other `COPY` lines

File: conf/php.ini (hunk 2 above)
Comment (gsc-dev) [inline conf/php.ini:7]: image

File: conf/php.ini (hunk 1 above)
Comment (gsc-dev) [inline conf/php.ini:1]: Hi @markoshust, I have a question about php.ini

I am not fluent with php but Is there a reason why are we not using the settings suggested by Magento?

https://github.com/magento/magento2/blob/develop/php.ini.sample

File: conf/php.ini (hunk 2 above)
Comment (markshust) [inline conf/php.ini:7]: Thank you, I misread the documentation. Apparently this line is still valid for PHP 5.6. I'll make sure it's only added to the 5.6 images then.

File: conf/php.ini (hunk 1 above)
Comment (markshust) [inline conf/php.ini:1]: Yes, I believe that file is only for standard php -- not php-fpm. There's not much going on in that php.ini file that would make any difference anyways. 64MB memory also seems really puny 😷

**Pull request closed without merging**